src/components/Form/CrossPodWriteForm.jsx

```diff
@@ -97,19 +77,11 @@ const CrossPodWriteForm = () => {
       statusType="Upload status"
       defaultMessage="To be uploaded..."
     >
-      <form onSubmit={handleCrossPodUpload} autoComplete="off">
+      <form onSubmit={handleFormSubmission} autoComplete="off">
         <div style={formRowStyle}>
-          <label htmlFor="cross-upload-doc">Upload document to username: </label>
+          <p>Upload document to {person}: </p>
```

Comment:
Should have a fallback for the value of person if person from the SelectUserContext.

Reply:
But then again, think it might be better if this remains an input field for username like before. We could have a line which shows the person's name above the input field if a person has been selected

Reply:
I removed the username input component because it's unused. We don't upload to usernames, we upload to Pod URLs. Whatever we input, it needs to map to a Pod URL. Usernames don't map to pod URLs any better than human names do, so why not just use their human name? We could have this be a dropdown that lets case managers select from their clients list, but that's a larger UI overhaul.